forge-game/src/main/java/forge/game/staticability/StaticAbilityContinuous.java

```diff
@@ -422,26 +422,30 @@ public static CardCollectionView applyContinuousAbility(final StaticAbility stAb
             if (params.containsKey("AddAllCreatureTypes")) {
                 addAllCreatureTypes = true;
             }
-            if (params.containsKey("RemoveSuperTypes")) {
-                remove.add(RemoveType.SuperTypes);
-            }
-            if (params.containsKey("RemoveCardTypes")) {
-                remove.add(RemoveType.CardTypes);
-            }
-            if (params.containsKey("RemoveSubTypes")) {
-                remove.add(RemoveType.SubTypes);
-            }
-            if (params.containsKey("RemoveLandTypes")) {
-                remove.add(RemoveType.LandTypes);
-            }
-            if (params.containsKey("RemoveCreatureTypes")) {
-                remove.add(RemoveType.CreatureTypes);
-            }
-            if (params.containsKey("RemoveArtifactTypes")) {
-                remove.add(RemoveType.ArtifactTypes);
-            }
-            if (params.containsKey("RemoveEnchantmentTypes")) {
-                remove.add(RemoveType.EnchantmentTypes);
+
+            // overwrite doesn't work without new value (e.g. Conspiracy missing choice)
+            if (addTypes == null || !addTypes.isEmpty()) {
```

Comment:
Isn't this wrong? What if something would just remove types without adding them? Wouldn't that break now?

Reply:
if it would only remove and not overwrite like _Alpine Moon_ addTypes will be null

Reply:
Hm, I somehow don't like the difference between `addTypes` not used, and `addTypes` being empty 🤔